x-pack/plugins/security_solution/server/connectors/sentinelone/sentinelone.ts

```diff
@@ -164,9 +167,26 @@ export class SentinelOneConnector extends SubActionConnector<
       },
       responseSchema: SentinelOneKillProcessResponseSchema,
     });
+
+    // write to .endpoint-actions index info about parentTaskId
```

Comment:
we could decide what and when to write to our "eventlog"-type of indices in order to track response actions properly (for example SentinelOne returns the `parentTaskId` prop, which is required to query for the action status). We can do it only after the action is taken, so having the connector registered in the Security solution makes it easier for us as we have access to all the utils here